_sass/_components/blog-index.scss

```diff
@@ -0,0 +1,111 @@
+// Blog index
```

Comment:
Yes, RSS moved to buttons.scss in c134cab, post list and pagination made into their own SCSS files in 3770842.